datadog/dogstatsd/base.py

```diff
@@ -160,10 +173,12 @@ def __enter__(self):
 
         def __exit__(self, type, value, traceback):
             # Report the elapsed time of the context manager.
-            self.statsd.timing(self.metric, time() - self.start,
-                               self.tags, self.sample_rate)
+            elapsed = time() - self.start
+            use_ms = self.statsd.use_ms or self.use_ms
```

Comment:
I realized after writing this that if the decorator had it, then it had to be passed into here. I think the logic on this statement probably needs to be rethough (likely: if `self.use_ms` is not explicitly passed in, then default to `self.statsd.use_ms`).